tests/projects/test_entry_point.py

```diff
@@ -127,36 +139,52 @@ def test_params():
 
     user_3 = {"alpha": 0.004, "gamma": 0.89}
     expected_final_3 = {"alpha": '0.004', "l1_ratio": '0.1', "l2_ratio": '0.0003',
-                        "random_str": "hello"}
+                        "random_str": "hello", "args": "--earlystopping"}
     expected_extra_3 = {"gamma": "0.89"}
     final_3, extra_3 = entry_point.compute_parameters(user_3, None)
     assert expected_extra_3 == extra_3
     assert expected_final_3 == final_3
 
     user_4 = {"alpha": 0.004, "l1_ratio": 0.0008, "random_str_2": "hello"}
     expected_final_4 = {"alpha": '0.004', "l1_ratio": '0.0008', "l2_ratio": '0.0003',
-                        "random_str": "hello"}
+                        "random_str": "hello", "args": "--earlystopping"}
     expected_extra_4 = {"random_str_2": "hello"}
     final_4, extra_4 = entry_point.compute_parameters(user_4, None)
     assert expected_extra_4 == extra_4
     assert expected_final_4 == final_4
 
     user_5 = {"alpha": -0.99, "random_str": "hi"}
     expected_final_5 = {"alpha": '-0.99', "l1_ratio": '0.1', "l2_ratio": '0.0003',
-                        "random_str": "hi"}
+                        "random_str": "hi", "args": "--earlystopping"}
     expected_extra_5 = {}
     final_5, extra_5 = entry_point.compute_parameters(user_5, None)
     assert expected_final_5 == final_5
     assert expected_extra_5 == extra_5
 
     user_6 = {"alpha": 0.77, "ALPHA": 0.89}
     expected_final_6 = {"alpha": '0.77', "l1_ratio": '0.1', "l2_ratio": '0.0003',
-                        "random_str": "hello"}
+                        "random_str": "hello", "args": "--earlystopping"}
     expected_extra_6 = {"ALPHA": "0.89"}
     final_6, extra_6 = entry_point.compute_parameters(user_6, None)
     assert expected_extra_6 == extra_6
     assert expected_final_6 == final_6
 
+    user_7 = {"alpha": 0.004, "gamma": 0.89, "args": ""}
+    expected_final_7 = {"alpha": '0.004', "l1_ratio": '0.1', "l2_ratio": '0.0003',
+                        "random_str": "hello", "args": "''"}
+    expected_extra_7 = {"gamma": "0.89"}
+    final_7, extra_7 = entry_point.compute_parameters(user_7, None)
+    assert expected_extra_7 == extra_7
+    assert expected_final_7 == final_7
+
+    user_8 = {"alpha": 0.004, "gamma": 0.89, "args": "-early-stopping"}
+    with pytest.raises(ExecutionException):
+        entry_point.compute_parameters(user_8, None)
+
+    user_9 = {"alpha": 0.004, "gamma": 0.89, "args": "early-stopping"}
```

Comment:
This test fails both because there are no hyphens before the argument and because there is a hyphen in the middle of the argument name. Since the test above already checks for failure due to the hyphen being in the middle of the argument name, can we make this change so we're only testing for missing hyphens before the argument?

```suggestion
    user_9 = {"alpha": 0.004, "gamma": 0.89, "args": "earlystopping"}
```

Reply:
Makes sense